O/oxigraph_server/build_tarballs.jl

```diff
@@ -35,9 +42,10 @@ products = Product[
 
 # Dependencies that must be installed before this package can be built
 dependencies = Dependency[
-    Dependency("OpenSSL_jll"; compat="1.1.10"),
+    Dependency("CompilerSupportLibraries_jll"),
+    Dependency("Libiconv_jll"),
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-                compilers=[:c, :rust], preferred_gcc_version=v"7", lock_microarchitecture=false, julia_compat="1.6")
+    julia_compat="1.6", lock_microarchitecture=false)
```

Comment:
You don't need `lock_microarchitecture` if you aren't building
```suggestion
    julia_compat="1.6")
```